systems/controllers/bodySpatialMotionPDmex.cpp

```diff
@@ -0,0 +1,80 @@
+#include "controlUtil.h"
+#include "drakeUtil.h"
+#include "drakeGeometryUtil.h"
+#include "mex.h"
+
+using namespace Eigen;
+using namespace std;
+
+void mexFunction(int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[])
+{
+  if(nrhs != 11)
```

Comment:
Should be a space after if before the open paren.  Also, the curly bracket should open on the same line.